nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/TagServiceImpl.java

```diff
@@ -45,6 +48,16 @@ public void start(ComponentContext context) {
     }
 
     @Override
+    public int getApplicationStartedOrder() {
+        // should deploy before repository service
```

Comment:
Add the reason: "because the tag service is indirectly used (through a listener) by the repository init handlers".